app/assets/stylesheets/components/_offices_table.scss

```diff
@@ -0,0 +1,27 @@
+.offices-table {
+  width: 100%;
+
+  .offices-table__address {
```

Comment:
Well spotted. The same thing has been done in _advisers_table.scss, I may fix there too. Asking the authors.